irksome/stage.py

```diff
@@ -271,44 +260,26 @@ def getFormStage(F, butch, u0, t, dt, bcs=None, splitting=None, vandermonde=None
     for bc in bcs:
         bcarg = as_ufl(bc._original_arg)
 
-        gblah_cur = []
-
         if bc in bc_constraints:
             bcparams, bclower, bcupper = bc_constraints[bc]
```

Comment:
`bcparams`, `bclower`, etc seem unused now?  The idea of this bit is that we need to be able to pass in special parameters to project the boundary condition expression into the finite element space subject to bounds constraints (e.g. using a VI solver).  It looks like this capability just disappeared.